src/controllers/words.js

```diff
@@ -74,11 +80,16 @@ export const getWords = async (req, res, next) => {
       query = searchEnglishRegexQuery(regexKeyword);
       words = await searchWordUsingEnglish({ query, ...searchQueries });
     } else {
+      const regularSearchFunction = wordClass ? searchIgboTextWithWordClass({
```

Comment:
Rename this to `regularIgboSearchQuery`. Your current variable name is misleading, it's technically not a function.

Reply:
Sorry about the re-review request I missed this comment! :)